packages/lb-components/src/utils/index.ts

```diff
@@ -130,10 +130,10 @@ export const getRectPointCloudBox = async (
     ...getBoundingRect(tmpPoints),
     valid,
     trackID,
-    imageName: mappingData.url,
+    path: mappingData.path,
```

Comment:
The annotation results still need to be harmonized with the client's

The `path` need to changed to `imageName`